### PR TITLE
 Refresh & AMS Release 1

### PR DESCRIPTION
**Pull request by infojunkie**
This is a WIP PR of all tickets for Refresh & AMS Release 1, until we are clear to push back to `master`.

### CODE REVIEW

File: src/web/themes/custom/workbc/workbc.theme [hunk 1]
```diff
@@ -72,6 +72,25 @@ function workbc_preprocess_paragraph(&$variables) {
...
 
+  if ($variables['paragraph']->bundle() == "content_interactive_map") {
+    $variables['region_info'] = ssotRegionInfo();  
+    $paragraph = $variables['paragraph'];
+    foreach ($variables['region_info'] as $key => $region) {
+      if ($region['key'] == "british_columbia") {
+        continue;
+      }
+      $field_key = substr("field_link_". $region['key'], 0, 32);
+      $test = "field_link_cariboo";
```
Comment (infojunkie) [inline src/web/themes/custom/workbc/workbc.theme:83]: @rjkuyvenhoven is this variable needed?

File: src/web/themes/custom/workbc/scss/layout/_footer.scss [hunk 2]
```diff
@@ -68,7 +68,8 @@
   }
 
   &.footer__social-link--twitter {
-    background-image: url(../assets/images/social-icon-twitter.svg);
+    background-image: url(../assets/images/social-icon-x.svg);
```
Comment (infojunkie) [inline src/web/themes/custom/workbc/scss/layout/_footer.scss:71]: @rjkuyvenhoven can we replace with `social-icon-x-color` instead ? and thus completely remove `social-icon-x`.

Comment (rjkuyvenhoven) [reply src/web/themes/custom/workbc/scss/layout/_footer.scss:71]: one is used in the share links and the other in the footer.  The share link classes specify a size, while the footer classes don't, except for the X class. However, it specifies a different size than is used in the share links.

File: src/web/themes/custom/workbc/workbc.theme (hunk 1 above)
Comment (rjkuyvenhoven) [reply src/web/themes/custom/workbc/workbc.theme]: Not needed and has been removed.

File: src/web/themes/custom/workbc/scss/layout/_footer.scss (hunk 2 above)
Comment (infojunkie) [reply src/web/themes/custom/workbc/scss/layout/_footer.scss:71]: alright let's keep it as is then

**Pull request merged**